styles.css

```diff
@@ -306,12 +318,10 @@ article {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  outline: none;
```

Comment:
Nice job catching these, I feel like you can end up with a lot of extra css when you are trying to nail down what is usually casuing the bahavior you are seeing.